aggregator/src/aggregator.rs

```diff
@@ -160,48 +149,44 @@ impl<
             debug!(target: "aleph-aggregator", "Aggregation already started for block hash {:?}, ignoring.", hash);
             return;
         }
-        self.multicast
-            .start_multicast(SignableHash::new(hash))
-            .await;
+        if let Some(multisigned) = self.rmc_service.start_rmc(hash) {
+            self.multisigned_events.push_back(multisigned);
+        }
     }
 
     async fn wait_for_next_signature(&mut self) -> IOResult {
```

Comment:
Explanation:
The interface of `rmc` has changed. 
Previous interface which treated obtaining multisignatures from rmc as an `async` process:
* send network messages (signed or multisigned hashes) to / from `rmc` by channels
* start the rmc protocol by calling a `void` function
* obtain `multisigned` hashes by polling on `next_multisigned_hash()`
Current interface, based on the idea that multisignatures can be created only after appending a single signature (obtaining multisigs is not `async` anymore):
* pass network messages to `rmc` by calling `process_message` which may return a `multisigned` hash
* receive messages from rmc for network by polling on `next_message`
* start rmc protocol by calling a function which may return `multisigned`
The new `wait_for_next_signature` was adjusted to match the new interface of `rmc` in a way that the interface of aggregator was preserved.